### ISSUE TITLE
[NEW] Donner la possibilité aux médiateurs d'embed des activités dans ses publications libres

### ISSUE THREAD

**Issue by larnoult** [parlemonde/1village#432]
Lorsqu'un médiateur publie un contenu libre, lui donner la possibilité de coller dans un bloc de type texte l'URL d'une activité (dans l'exemple ci-dessous https://1v.parlemonde.org/activite/335) 

![Image](https://user-images.githubusercontent.com/20889221/191346412-8e023d44-6db5-476e-9f6e-27158ae067e4.png)

Une fois publiée, cette URL est remplacée par la carte de prévisualisation de l'activité correspondante 

![Image](https://user-images.githubusercontent.com/20889221/191346433-d677d91c-1fe9-4096-b3d9-adbe1f9889dd.png)

**Comment by larnoult** [comment]
Je ne trouve pas ça très satisfaisant. Je fais deux contre-propositions : 
* soit on laisse tomber le fait que seul le médiateur peut réaliser l'embed (et le comportement est le même pour les classes). 
* soit, pour que seul le médiateur puisse le réaliser, je propose qu'il doive rentrer l'url en utilisant le bouton "h5p" et en collant le lien dans l'invite 
![image](https://user-images.githubusercontent.com/20889221/198052813-26feeedf-83a6-4d62-a0b0-31c3af73bdc9.png)
(seuls les utilisateurs au droit >= médiateur ont accès au bouton).

**Comment by yudino** [comment]
@larnoult On va rester sur la première proposition pour le moment. Je te met la branche sur staging pour testing.

**Comment by larnoult** [comment]
@yudino  
De mon côté, cela ne fonctionne pas. Le bouton choisir reste grisé. 

![Image](https://user-images.githubusercontent.com/20889221/203381589-0aaaf905-c2e9-4e9d-91aa-2048198d7d1f.png)

**Comment by yudino** [comment]
@larnoult Oui, c'est normal comme on avait dit l'option 1 et du coup pas l'ajout dans le h5p.

**Comment by larnoult** [comment]
Indeed ! Désolé, je n'avais pas bien suivi. Du coup, cela fonctionne. Néanmoins, je pense qu'il faudrait enlever l'url de l'aperçu de la carte résumée (c'est assez confusionnant). Par ex : 

![Image](https://user-images.githubusercontent.com/20889221/203384006-c2cdbafb-4c22-4006-b729-d5d51738ddd7.png)

**Comment by yudino** [comment]
@larnoult Je vais regarder car à ma connaissance, je n'ai pas eu ce souci dans l'activité.

**Comment by yudino** [comment]
@larnoult Sauf si ce n'est le bon comportement, la fonctionnalité que tu demandes fonctionne uniquement à l'étape 1 et pas quand on met le lien dans l'extrait à l'étape 2.
https://www.loom.com/share/a9041b7090d045bca45a5f82423d9464
